crates/db_schema/src/impls/local_user.rs

```diff
@@ -216,6 +217,45 @@ impl LocalUser {
   }
 }
 
+/// Adds some helper functions for an optional LocalUser
+pub trait LocalUserOptionHelper {
```

Comment:
These functions are only run by LocalUser anyway, so its fine to have the trait here.